src/dto/derived-hourly-value.dto.ts

```diff
@@ -1,40 +1,41 @@
-import { IsValidCode } from '@us-epa-camd/easey-common/pipes';
 import {
   IsNumber,
   IsOptional,
   IsString,
-  ValidationArguments,
+  Matches,
+  Max,
+  Min,
 } from 'class-validator';
-import { ParameterCode } from '../entities/parameter-code.entity';
-import { ImportCodeErrorMessage } from '../utils/validator.const';
-import { ModcCode } from '../entities/modc-code.entity';
-import { OperatingConditionCode } from '../entities/operating-condition-code.entity';
-import { FuelCode } from '../entities/fuel-code.entity';
+import { FORMULA_ID_REGEX } from '../constants/regex-list';
 
 export class DerivedHourlyValueBaseDTO {
   @IsString()
-  @IsValidCode(ParameterCode, {
-    message: (args: ValidationArguments) => {
-      return ImportCodeErrorMessage(args.property, args.value);
-    },
-  })
+  // @IsValidCode(ParameterCode, {
+  //   message: (args: ValidationArguments) => {
+  //     return ImportCodeErrorMessage(args.property, args.value);
+  //   },
+  // })
   parameterCode: string;
 
   @IsNumber()
   @IsOptional()
+  @Min(0)
+  @Max(9999999999.9999)
   unadjustedHourlyValue: number;
 
   @IsNumber()
   @IsOptional()
+  @Min(0)
+  @Max(9999999999.9999)
   adjustedHourlyValue: number;
 
   @IsString()
   @IsOptional()
-  @IsValidCode(ModcCode, {
-    message: (args: ValidationArguments) => {
-      return ImportCodeErrorMessage(args.property, args.value);
-    },
-  })
+  // @IsValidCode(ModcCode, {
+  //   message: (args: ValidationArguments) => {
+  //     return ImportCodeErrorMessage(args.property, args.value);
+  //   },
+  // })
   modcCode: string;
 
   @IsString()
```

Comment:
Format validation needed for monitoringSystemId

Reply:
Updated